### PR TITLE
add cacheKey null check 

### PR DESCRIPTION
**Pull request by HHChaos**
when using LottieView.SetAnimationFromJsonAsync() or other methods that quote LottieComposition.Factory.FromJsonStringAsync(),it will cause a null exception

### CODE REVIEW

File: LottieUWP/LottieDrawable.cs [hunk 1]
```diff
@@ -68,6 +68,43 @@ public LottieDrawable()
             Unloaded += UserControl_Unloaded;
         }
 
+        public CanvasCommandList GetCanvasImage(ICanvasResourceCreator resourceCreator, float scaleX, float scaleY)
```
Comment (azchohfi) [inline LottieUWP/LottieDrawable.cs]: This method feels like a different necessity. I like it, but does not feels like it should be on the same PR.

File: LottieUWP/LottieDrawable.cs [hunk 2]
```diff
@@ -68,6 +68,43 @@ public LottieDrawable()
...
+        {
+            lock (this)
+            {
+                var commandList = new CanvasCommandList(resourceCreator);
+                using (var session = commandList.CreateDrawingSession())
+                {
+                    var width = _composition.Bounds.Width * scaleX;
+                    var height = _composition.Bounds.Height * scaleY;
+                    if (_bitmapCanvas == null || _bitmapCanvas.Width < width || _bitmapCanvas.Height < height)
```
Comment (azchohfi) [inline LottieUWP/LottieDrawable.cs]: Why only when less than **<**, not different **!=**?

File: LottieUWP/LottieDrawable.cs [hunk 3]
```diff
@@ -68,6 +68,43 @@ public LottieDrawable()
...
+                        {
+                            return null;
+                        }
+
+                        _matrix.Reset();
+                        _matrix = MatrixExt.PreScale(_matrix, scaleX, scaleY);
+                        _compositionLayer.Draw(_bitmapCanvas, _matrix, _alpha);
+                        LottieLog.EndSection("Drawable.Draw");
+                    }
+
```
Comment (azchohfi) [inline LottieUWP/LottieDrawable.cs]: Empty line.

Comment (HHChaos) [reply LottieUWP/LottieDrawable.cs]: Thanks! I will remove it.

File: LottieUWP/LottieDrawable.cs (hunk 1 above)
Comment (HHChaos) [reply LottieUWP/LottieDrawable.cs]: Sorry, I should delete this method before I create this PR.
I use it to render image, but it not work well on some times(Issues #37)

File: LottieUWP/LottieDrawable.cs (hunk 2 above)
Comment (HHChaos) [reply LottieUWP/LottieDrawable.cs]: Because I'm using this condition to make sure that the image doesn't get blurry when I zoom in

**Pull request merged**